app/customer_routes.py

```diff
@@ -0,0 +1,84 @@
+from flask import Blueprint, jsonify, request
+from app import db
+from app.models.customer import Customer
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+@customers_bp.route("", methods=["GET"])
+def get_customers():
+    customers = Customer.query.all()
+    customers_response = [customer.to_json() for customer in customers]
+    return jsonify(customers_response), 200
+
+@customers_bp.route("", methods=["POST"])
+def post_customers():
+    request_body = request.get_json()
+    if "name" not in request_body:
+        return {"details": "Request body must include name."}, 400 
+    elif "postal_code" not in request_body:
+        return {"details": "Request body must include postal_code."}, 400
+    elif "phone" not in request_body:
+        return {"details": "Request body must include phone."}, 400
```

Comment:
This would be great as a helper function, and maybe we can even figure out a way to have only one return statement that dynamically changes out the request body attribute.